ui/mirage/serializers/agent.js

```diff
@@ -0,0 +1,10 @@
+import ApplicationSerializer from './application';
+
+export default ApplicationSerializer.extend({
+  keyForAttribute(str) {
+    if (str === 'config' || str === 'member') {
+      return str;
+    }
+    return ApplicationSerializer.prototype.keyForAttribute.apply(this, arguments);
+  },
```

Comment:
The alternative here could be turning `keyCase` on `ApplicationSerializer` into a more generic API.

```
const keyCase = (str, config, customConfig) => {
  switch (config.type):
    case overridePascalCase:
      return str;
    case customConfig:
      return CustomConfig();
    default:
      return defaultLogic();
}
```

Seems like overkill, but I think its still a worthwhile thought experiment to how we think about DRY v WET, readability, and standards.

Reply:
`keyCase` is used by every other factory, I'm not sure there would be much benefit in modifying it just for this case. The way it's implemented right now, we're using inheritance to expand on a default parent behaviour, which seems reasonable to me?

Reply:
Gotcha. The other factories that need to further abstract against the `keyCase` logic also follow this pattern so I see that its inline with what we're currently doing.